React_Fundamentals/src/common/ProgressBarForm.tsx

```diff
@@ -0,0 +1,22 @@
+import { useState } from "react"
+import ProgressBar from "./ProgressBar"
+
+export const ProgressBarForm = () => {
+    const [value, setValue] = useState(0);
+
+    return <form>
+        <fieldset>
+            <legend>Progress bar</legend>
+            <input 
+                type="number"
+                pattern="\d+"
+                min={0}
+                max={100}
+                step={1}
+                onChange={(e) => {
+                setValue(+e.target.value)
```

Comment:
this is not quite correct.. here you can implement validation to avoid entering numbers over 100